src/facade/reply_builder.h

```diff
@@ -103,6 +104,13 @@ class SinkReplyBuilder {
   void SendError(ErrorReply error);
   virtual void SendProtocolError(std::string_view str) = 0;
 
+  virtual void StartCollection(unsigned len, CollectionType ct) = 0;
```

Comment:
why do we need these functions here?

Reply:
Because we need the `CaptureReplyBuilder` to use those based on the `protocol`. Before we had `SendMGetResponse` which Vlad removed.